WireAnalytics/Sources/WireAnalytics/Events/Segmentation/Predefined/Segmentation+Removed.swift

```diff
@@ -26,13 +26,21 @@ extension AnalyticsEvent.Segmentation {
 
     enum Removed {
 
+        // https://wearezeta.atlassian.net/browse/WPB-12199?focusedCommentId=132080
+        @available(*, deprecated, message: "Use `AnalyticsEvent.Segmentation.Conversation.ConversationType`.")
+        enum ConversationType: String {
+            case group
+            case oneOnOne = "one_on_one"
+            case unknown
```

Comment:
What is an example of the `unknown` conversation type?

Reply:
Basically anything other than group or one to one, such as `self` or even `invalid`. But this is an old segmentation that should be phased out.